WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -34,27 +34,15 @@ final class TotalsViewModel: ObservableObject, TotalsViewModelProtocol {
 
     @Published private(set) var connectionStatus: CardReaderConnectionStatus = .disconnected
 
-    @Published var formattedCartTotalPrice: String?
-    @Published var formattedOrderTotalPrice: String?
-    @Published var formattedOrderTotalTaxPrice: String?
+    @Published private(set) var formattedCartTotalPrice: String?
+    @Published private(set) var formattedOrderTotalPrice: String?
+    @Published private(set) var formattedOrderTotalTaxPrice: String?
```

Comment:
I don't think we can properly test these in the current state, so I left it for a future PR:
- Populating values for these properties its done via `TotalsViewModel.updateOrder()` which is private, and is invoked as a side-effect of `TotalsViewModel.syncOrder()`. It seems an indication that this also should be moved somewhere else, perhaps some sort of `OrderUpdater` or a small struct that contains all the transaction data, then is passed into the view model.
- Even if we could test the invocation of `updateOrder()`,  the result would tell us if the `CurrencyFormatter` works as expected, which is tested extensively on its own, and shouldn't be part of this.